examples/README.md

```diff
@@ -49,6 +49,17 @@ $ # In another terminal run the client
 $ mvn exec:java -Dexec.mainClass=io.grpc.examples.helloworld.HelloWorldClient
 ```
 
+## Bazel
+
+If you prefer to use Bazel:
+```
+(With Bazel v0.4.5 or above.)
+$ # Run the server:
+$ bazel run //:hello-world-server
+$ # In another terminal run the client
+$ bazel run //:hello-world-client
```

Comment:
This doesn't work, because you can only have one `bazel` in a workspace running at a time. Instead do the following?

```
$ bazel build :hello-world-server :hello-world-client
$ # Run the server:
$ bazel-bin/hello-world-server
$ # In another terminal run the client
$ bazel-bin/hello-world-client
```